examples/plugins/joinargs-plugin.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "JoinArgsPlugin",
+    "integrationPoints": [
+        {
+            "interface": "CLICommand",
```

Comment:
This I would consider as one option, where a rpc plugin standardizes the output around an arbitary locally invoked binary